sdk/src/main/java/io/opentelemetry/sdk/trace/SpanProcessor.java

```diff
@@ -48,4 +48,12 @@
 
   /** Called when {@link TracerSdkRegistry#shutdown()} is called. */
   void shutdown();
+
+  /**
+   * Exports all ended spans that have not yet been exported.
+   *
+   * <p>This method is called synchronously on the execution thread, and should not throw exceptions
+   * or block the execution thread. execution thread.
```

Comment:
I am confused about this. I think the whole purpose is to ensure flush so it will block, maybe add an issue to clarify this behavior on the specs

Reply:
If it is not blocking maybe a name like initiateFlush or tryFlush would be better.

Reply:
Typo: doubled the last 2 words

Reply:
The current implementation is blocking. But when I wrote the Javadoc I had the version with timeout in mind. I will rewrite this to clarify the intention of the method!